src/security/ServerOptions.cc

```diff
@@ -50,7 +54,18 @@ Security::ServerOptions::parse(const char *token)
     }
 
     // parse the server-only options
-    if (strncmp(token, "dh=", 3) == 0) {
+    if (strncmp(token, "context=", 8) == 0) {
+#if USE_OPENSSL
+        staticContextSessionId = SBuf(token+8);
+        if (staticContextSessionId.length() > SSL_MAX_SSL_SESSION_ID_LENGTH) {
+            debugs(83, DBG_CRITICAL, "FATAL: Option 'context=' value is too long. Maximum " << SSL_MAX_SSL_SESSION_ID_LENGTH << " characters.");
```

Comment:
The value of this parameter is a security secret, effectively a password for the session. So causing it to be logged is not a good idea. A small amount of "duplication" by naming just the option in text seems a reasonable tradeoff for protecting that.